alot/commands/envelope.py

```diff
@@ -146,8 +146,8 @@ def apply(self, ui):
                 ui.apply_command(globals.FlushCommand())
                 ui.apply_command(commands.globals.BufferCloseCommand())
             except DatabaseError as e:
-                logging.error(e.message)
-                ui.notify('could not index message:\n%s' % e.message,
+                logging.error(str(e))
+                ui.notify('could not index message:\n%s' % str(e),
```

Comment:
I think logging automatically calls str on its argument. At least for string interpolation it is not neccessary.